packages/agents/README.md

```diff
@@ -0,0 +1,112 @@
+# 🤗 Hugging Face Agents
```

Comment:
```suggestion
# 🤗 Hugging Face Agents.js
```

Reply:
btw you can also add something like this to the doc IMO: https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/transformers/diagram.png